src/main/Validator.ts

```diff
@@ -119,7 +119,7 @@ const configDataSchemaV3 = Joi.object<ConfigV3>({
     useSpellChecker: Joi.boolean().default(true),
     enableHardwareAcceleration: Joi.boolean().default(true),
     autostart: Joi.boolean().default(true),
-    spellCheckerLocales: Joi.array().items(Joi.string().regex(/^[a-z]{2}-[A-Z]{2}$/)).default([]),
+    spellCheckerLocales: Joi.array().items(Joi.string().regex(/^[A-Za-z-]+$/)).default([]),
```

Comment:
we should limit the size, I don't think there is more than 5 or 6 chars, but anyway it is a standard so we can probably check on that.
Either that or accept any string and lookup the values later in the app

Reply:
actually, I don't really see much value on this regex validation: a string is just fine